### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by alix-mak**

### CODE REVIEW

**Review by liuda-chuzdiuk — APPROVED** [review]

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,22 @@
...
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_decline_lower_five_percent(mocked_get_exchange_rate_prediction):
+    mocked_get_exchange_rate_prediction.return_value = 1
+    assert cryptocurrency_action(1.5) == "Sell all your cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_difference_is_not_that_much(mocked_get_exchange_rate_prediction):
+    mocked_get_exchange_rate_prediction.return_value = 1
+    assert cryptocurrency_action(1.05) == "Do nothing"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:20]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]
